modules/ROOT/pages/wifi.adoc

```diff
@@ -0,0 +1,48 @@
+
+== How do I add the missing wifi-firmwares?
+
+Some wifi-firmwares will be split into sub-packages in Fedora 39 and Fedora 40.
+Fedora Cores OS will keep them in until Fedora 41, but display a
+warning message in the console if NetworkManager-wifi is layered without
+any other wifi firmware packages having a layering request.
+
+If NetworkManager-wifi is a requested layered package, a warning is
+displayed to encourage users to also add a wifi-firmware.
+
+.Install thre required firmware for your hardware:
+[source, text]
+----
+$ sudo rpm-ostree install iwlwifi-dvm-firmware --allow-inactive
+----
+
+If you don't know what firmware to install you can install
+all the wireless firmwares available in Fedora.
+Please note this approach is un-optimal as it will add a lot
+of uneeded packages on your system.
+.Install all the wifi firmwares
+----
+$ sudo rpm-ostree install --allow-inactive \
+atheros-firmware \
+b43-fwcutter \
+b43-openfwwf \
+brcmfmac-firmware \
+iwlegacy-firmware \
+iwlwifi-dvm-firmware \
+iwlwifi-mvm-firmware \
+libertas-firmware \
+mt7xxx-firmware \
+nxpwireless-firmware \
+realtek-firmware \
+tiwilink-firmware \
+atmel-firmware \
+bcm283x-firmware \
+zd1211-firmware
+----
```

Comment:
remove `bcm283x-firmware` I don't think it exists?

```
No match for argument: bcm283x-firmware
Error: Unable to find a match: bcm283x-firmware
```

Also suggest adding `-y` and making spacing more attractive:

```suggestion
$ sudo rpm-ostree install -y --allow-inactive \
    atheros-firmware     \
    b43-fwcutter         \
    b43-openfwwf         \
    brcmfmac-firmware    \
    iwlegacy-firmware    \
    iwlwifi-dvm-firmware \
    iwlwifi-mvm-firmware \
    libertas-firmware    \
    mt7xxx-firmware      \
    nxpwireless-firmware \
    realtek-firmware     \
    tiwilink-firmware    \
    atmel-firmware       \
    bcm283x-firmware     \
    zd1211-firmware
----
```